src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,113 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationServiceImpl service;
+    private User user;
+
+    @BeforeAll
+    static void beforeAll() {
+        service = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+        user.setId(474797L);
+        user.setLogin("userLogin47");
+        user.setPassword("password1111");
+        user.setAge(25);
+    }
+
+    @Test
+    void register_UserValid_Ok() {
+        StorageDao storageDao = new StorageDaoImpl();
+        assertNotNull(service.register(user));
+        assertNotNull(storageDao.get(user.getLogin()));
+    }
+
+    @Test
+    void register_UserNullValue_NotOk() {
+        service.register(null);
+        assertThrows(InvalidUserException.class, () -> service.register(null),
+                "User must not be null");
```

Comment:
Your assertion is placed after the method call that is expected to throw an exception. The `assertThrows` should be the one calling the `service.register(null)` method to correctly test exception handling.